src/wallet/wallet.cpp

```diff
@@ -3107,6 +3107,8 @@ CAmount CWallet::GetMinimumFee(unsigned int nTxBytes, const CCoinControl& coin_c
 
 DBErrors CWallet::LoadWallet(bool& fFirstRunRet)
 {
+    LOCK2(cs_main, cs_wallet);
```

Comment:
Looks good. You can remove the `LOCK(cs_wallet);` from further down in this function (currently L3118)

Reply:
Done in 09ee3faaf631fb78150c2725775c65ede66d3a77